### PR TITLE
Licenses feature

### PR DESCRIPTION
**Pull request by GhitaB**

### CODE REVIEW

File: eea/rdfmarshaller/browser/templates/license.pt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<tal:license tal:define="license_text python:view.license_text">
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/browser/templates/license.pt]: This file is not needed. You can render the script from the viewlet render() method

File: eea/rdfmarshaller/browser/viewlets.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                x for x in reg_licenses if x['license_id'] == license_id]
+
+            if len(licenses) == 0:
+                return None  # No license details for this license id
+
+            license = licenses[0]
+            license_url = license.get("license_url", "")
+            license_title = license.get("license_id", "")
+
+            text = json.dumps(
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/browser/viewlets.py]: This is not using the eea.rdfmarshaller properly. This is work that isn't yet commited, still in work by me and Alec, so it's fine for now, but will need refactoring.

File: eea/rdfmarshaller/controlpanel/license.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+from zope.component import getUtility
+from zope.interface import Interface
+from zope.interface import implementer
+from zope.schema.interfaces import IVocabularyFactory
+from zope.schema.vocabulary import SimpleTerm
+from zope.schema.vocabulary import SimpleVocabulary
+
+
+class ILicense(Interface):
+    license_id = schema.ASCIILine(
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/controlpanel/license.py]: Matybe rename field to ``id``

File: eea/rdfmarshaller/controlpanel/license.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        title=u"License Title",
+        description=u"Used to assign a license to a portal type."
+    )
+
+    license_text = schema.Text(
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/controlpanel/license.py]: Rename field to ``text``

File: eea/rdfmarshaller/controlpanel/license.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        title=u"License Text",
+        description=u"Human readable text of license"
+    )
+
+    license_url = schema.URI(
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/controlpanel/license.py]: rename field to ``url``

File: eea/rdfmarshaller/controlpanel/license.py [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    """ Define settings data structure for portal types - licenses """
+
+    rdfmarshaller_type_licenses = schema.Dict(
+        title=u"Licenses",
+        description=u"Choose licenses.",
+        key_type=schema.Choice(
+            title=u"Portal Type",
+            vocabulary="plone.app.vocabularies.ReallyUserFriendlyTypes"),
+        value_type=schema.Choice(
+            title=u"License Title",
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/controlpanel/license.py]: rename title to "License"

File: eea/rdfmarshaller/controlpanel/license.py [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+
+@implementer(IVocabularyFactory)
+class LicensesVocabulary(object):
+
+    def __call__(self, context):
+        registry = getUtility(IRegistry)
+        try:
+            licenses = registry[
+                "eea.rdfmarshaller.controlpanel.license.ILicenses"
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/controlpanel/license.py]: There's a constant defined for this. Anyway, those constant names should be shortened?

File: eea/rdfmarshaller/profiles/default/controlpanel.xml [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<object
+  name="portal_controlpanel"
+  xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+  i18n:domain="silvuple">
+
+  <configlet
+    title="Global Licenses"
```
Comment (tiberiuichim) [inline eea/rdfmarshaller/profiles/default/controlpanel.xml]: Rename to "Copyright License Definitions"

**Comment by GhitaB** [comment]
Updated based on your changes request.

**Comment by GhitaB** [comment]
Added tests, uninstall profile.

**Comment by GhitaB** [comment]
Copyright notice and Attribution text are now configurable, too.

**Pull request merged**